CHANGELOG

```diff
@@ -1,6 +1,8 @@
 2.7.2 (compared to 2.7.1)
 -----------------------------
 
+- preserve warning functions after call to pytest.deprecated_call
```

Comment:
Could you add a "Thanks Pieter Mulder for PR." to the end, as we usually do with the other entries. Thanks